src/renderer/families/tezos/DelegateFlowModal/steps/StepVerification.js

```diff
@@ -20,9 +20,11 @@ const StepVerification = ({
   }, [signTransaction, transitionTo]);
 
   // didMount
+  // this was done only on mount on v1, so I removed the
+  // dependencies to simulate that (val)
```

Comment:
(interesting to dig why this code is needed. I thought we had "useSignTransaction" for this)

Reply:
we can probably use `useSignTransaction`

Reply:
maybe it's diff level of things. we'll rediscuss in future